processFlows.dscp

```diff
@@ -34,12 +43,23 @@ token IssuedCert {
 pub fn initiate_cert || => |output: InitiatedCert| where {
   output.hydrogen_owner == sender,
   output.energy_owner != sender,
+  output.regulator != sender,
 }
 
 pub fn issue_cert |input: InitiatedCert| => |output: IssuedCert| where {
   output == input,
   input.hydrogen_owner == output.hydrogen_owner,
   input.energy_owner == output.energy_owner,
   output.hydrogen_owner != sender,
+  output.regulator!= sender,
```

Comment:
need to add a restriction:

`input.regulator == output.regulator`

Reply:
and on `revoke_cert`

Reply:
this has been added.